src/stream.rs

```diff
@@ -617,9 +617,10 @@ impl SendBuf {
     }
 
     /// Returns contiguous data from the send buffer as a single `RangeBuf`.
-    pub fn pop(&mut self, max_data: usize) -> Result<RangeBuf> {
+    pub fn pop(&mut self, max_data: u64) -> Result<RangeBuf> {
```

Comment:
I think `max_data` should stay as `usize` as its value depends on `left` and the size of the output buffer.

Reply:
`max_data` is from transport parameter or MAX_DATA frame which is variable int, so need to be `u64` I think.

Reply:
Oh I understand now. It's different from MAX_DATA... confused by name. I will make changes.

Reply:
Fixed at 5fffab9